loadtest/control/simulcontroller/controller.go

```diff
@@ -207,6 +207,11 @@ func getActionList(c *SimulController) []userAction {
 			run:       c.generateUserReport,
 			frequency: 0.0001,
 		},
+		{
+			name:      "SubmitClientPerformanceReport",
+			run:       submitPerformanceReport,
+			frequency: 0.0001,
```

Comment:
Is this enough for sending it regularly? Again, I don't know how the browser does it, but if it follows a fixed frequency, maybe we should do something like that here?

Reply:
The browser should send this once in every minute but from my tests locally it's being sent whenever I focus to tab. So it doesn't send the metrics while I'm not using it.